web/packages/teleport/src/teleportContext.tsx

```diff
@@ -195,4 +168,33 @@ class TeleportContext implements types.Context {
   }
 }
 
+export const falseFlags = {
```

Comment:
When defining an object literal, it is good to add a type to it, so that TS will show errors here instead of where the object is used.
  
```suggestion
export const falseFlags: types.FeatureFlags = {
```

Also maybe `disabledFeatureFlags` would be a better name?